libs/examples/src/highlight/highlight-term-array-example/highlight-term-array-example.html

```diff
@@ -0,0 +1,28 @@
+<input
+  type="text"
+  dtInput
+  value="sit,est,lacus massa"
+  #search
+  aria-label="Split terms by ','"
+/>
+<dt-highlight [term]="search.value.split(',')" caseSensitive>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi id ultrices
+  eros. Integer viverra tempor lacinia. Aliquam ac nibh consectetur, auctor
```

Comment:
You need to add this file declaration in `libs/examples/src/index.ts` as well as add a routing link in `apps/demos/src/app-routing.module.ts`

Reply:
Please don't forget to add the nav link to` app-routing.module.ts` and `nav-items.ts` as was done here: https://github.com/dynatrace-oss/barista/pull/2596/files#diff-ffeda48d19c6203493253e37af0053e5732bc3952ede6423d1f5544919939abc